packages/venia-ui/lib/components/Country/country.js

```diff
@@ -15,7 +16,14 @@ const Country = props => {
         }
     });
     const { countries, loading } = talonProps;
-    const { classes: propClasses, field, label, ...inputProps } = props;
+    const {
+        classes: propClasses,
+        field,
+        label,
+        translationId,
```

Comment:
Good choice here to take this in as a prop. Naming looks good, too, though I wonder what happens if we need more than one.